src/services/latex.js

```diff
@@ -27,6 +27,12 @@ var latexMathParser = (function() {
   var variable = letter.map(function(c) { return Letter(c); });
   var symbol = regex(/^[^${}\\_^]/).map(function(c) { return VanillaSymbol(c); });
 
+  // Compound commands e.g. \not\subset
+  var compound = regex(/^\\not\\[a-z]+/).then(function(cmp) {
```

Comment:
Yeah exactly. There's actually only one other one in mathquill: `\text\AA` (angstrom symbol) which is also currently broken and could be fixed by adding to this regex. Didn't want to overcomplicate it though for now.